src/arutils.py

```diff
@@ -437,6 +456,8 @@ def polyval2d(x, y, m):
         z += a * x**i * y**j
     return z
 
+def gauss_3deg(x,ampl,cent,sigm):
+    return ampl*np.exp(-1*(cent-x)**2/2/sigm**2)
```

Comment:
I think that we could (...perhaps should) return the integral of a Gaussian between neighboring pixels, instead of just to a Gaussian at the location of the pixel. I'm fairly confident that there's enough information available to account for the curvature of a Gaussian profile within a pixel, and this will affect the profile shape.

For the functional form using the same set of input parameters, see the `gfunc()` function on line 470 (just ignore the cons and tilt arguments).